vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -2,6 +2,12 @@ module SecureRooms
 
   class Occupancy < ActiveRecord::Base
 
+    include TextHelpers::Translation
```

Comment:
Probably not. Might have been a merge thing from when we were using it for problem descriptions. I'll remove it.